shared/src/question-generators/huffman-coding/huffmanCoding.ts

```diff
@@ -41,30 +41,39 @@ const translations: Translations = {
   en: {
     name: "Compute a Huffman-Coding",
     description: "Compute the Huffman-Coding of a given string",
-    text: 'Let "*{{0}}*" be {{1}}. What is a correct **Huffman-Coding** of this {{1}}?',
+    text: 'Let "*{{0}}*" be {{1}}. What is a correct **Huffman-Coding** of this {{1}}? Please do not consider the spaces.',
     prompt: "What is a possible Huffman-Coding?",
     textTable: `Suppose we have the following table, which represents how often each character appears in a given string:
 {{0}}
 What could be a correct **Huffman-Coding** for each char?`,
-    feedbackInvalid: "Can only contain 1 and 0",
+    feedbackInvalid: "Please only use the characters 0 and 1.",
     bottomtext:
       "Hints for the Huffman-Code: If you have to choose between nodes with the same weight, " +
       "first choose the one in whose subtree the alphabetically smaller character is contained." +
       " Also choose as the left node, the node with the smaller weight.",
+    multiInputText: `Suppose we have the following table, which represents how often each character appears in a given string:
+{{0}}
+What could be a correct **Huffman-Coding** for each character?
```

Comment:
Besser als Aufforderung formulieren: `Construct a correct Huffman-Code for the given string.`